### PR TITLE
added avatar and badges files

### PR DESCRIPTION
**Pull request by ajit-sharmaGH**

### CODE REVIEW

**Comment by JantuDeb** [comment]
> _No description provided._
Add concise description and preview for components

File: CSS/avatar.css [hunk 1]
```diff
@@ -1,3 +1,115 @@
...
+
+.larger {
+    height: 7rem;
+    width: 7rem;
+}
+
+.profile {
+    height: 7rem;
+    width: 7rem;
+}
```
Comment (JantuDeb) [inline CSS/avatar.css:64]: Try to use self-explanatory class names, here it is unclear whether larger, small is used for padding, margin or width, height.

File: CSS/avatar.css [hunk 2]
```diff
@@ -1,3 +1,115 @@
...
+    background-color: var(--iframe-bg-color);
+    display: inline-block;
+    height: 40vh;
+    border: none;
+}
+
+.iframe-container::-webkit-scrollbar {
+    height: 0px;
+    width: 0px;
+} */
```
Comment (JantuDeb) [inline CSS/avatar.css:92]: remove unused code from documentation.

File: CSS/avatar.css [hunk 3]
```diff
@@ -1,3 +1,115 @@
...
 @import url("base/color.css");
-@import url("base/nav.css");
\ No newline at end of file
+@import url("base/nav.css");
+.avatar-content {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    background-color: antiquewhite;
```
Comment (tusharkandpal) [inline CSS/avatar.css:9]: use color variable everywhere possible.. will help

**Review by JantuDeb — COMMENTED** [review]
Overall the code is good also I have suggested some changes you can look into this

File: CSS/avatar.css [hunk 4]
```diff
@@ -1,3 +1,115 @@
...
+    justify-content: center;
+    box-shadow: 1px 1px 1px 1px var(--dark-primary);
+    width: 70%;
+    overflow: hidden;
+    height: 22vh;
+    border-radius: 0.5rem;
+    background-color: var(--dark-primary-text);
+}
+
+.avatar img {
```
Comment (tusharkandpal) [inline CSS/avatar.css:39]: avoid using img tag directly

File: CSS/base/nav.css [hunk 5]
```diff
@@ -70,7 +71,6 @@ nav ul li a:active {
     width: 66%;
     transform: translateY(0%);
     transition: transform 0.5s ease-in;
-    /* z-index: 2; */
```
Comment (tusharkandpal) [inline CSS/base/nav.css:73]: remove this comment

**Review by tusharkandpal — CHANGES_REQUESTED** [review]
suggested few changes

**Comment by ajit-sharmaGH** [comment]
> suggested few changes

Thank you will do that

**Comment by ajit-sharmaGH** [comment]
> Overall the code is good also I have suggested some changes you can look into this

Thanks a lot, will do that

File: CSS/avatar.css (hunk 4 above)
Comment (ajit-sharmaGH) [reply CSS/avatar.css:39]: how should I use that can u suggest me the syntax ? I used it under avatar class as per my knowledge!

**Pull request merged**